src/Codeception/Lib/InnerBrowser.php

```diff
@@ -1244,7 +1244,7 @@ protected function debugResponse($url)
         $this->debugSection('Response Headers', $this->getRunningClient()->getInternalResponse()->getHeaders());
     }
 
-    protected function getResponseStatusCode()
+    public function getResponseStatusCode()
```

Comment:
In this case method name should be prefixed with `_` so it won't go to a tester class and would be accessible only from helpers.

Reply:
Like https://github.com/Codeception/Codeception/pull/4745/commits/8f4a04970dbcd46bd1a8d99d4197deea57b5302a ?